src/qml/QFieldSettings.qml

```diff
@@ -265,6 +256,54 @@ Page {
                       }
                   }
 
+                  GridLayout {
+                      Layout.fillWidth: true
+                      Layout.leftMargin: 20
+                      Layout.rightMargin: 20
+                      Layout.topMargin: 5
+                      Layout.bottomMargin: 40
+
+                      columns: 1
+                      columnSpacing: 0
+                      rowSpacing: 5
+
+                      visible: platformUtilities.capabilities & PlatformUtilities.AdjustBrightness
+
+                      Label {
+                          Layout.fillWidth: true
+                          text: qsTr('Dim screen when idling')
+                          font: Theme.defaultFont
+
+                          wrapMode: Text.WordWrap
+                      }
+
+                      Label {
+                          Layout.fillWidth: true
+                          text: qsTr('Time of inactivity in seconds before the screen brightness get be dimmed to preserve battery.')
+
+                          font: Theme.tipFont
+                          color: Theme.gray
+                          wrapMode: Text.WordWrap
+                      }
+
+
+                      QfSlider {
+                        id: slider
+                        value: settings ? settings.value('dimTimeoutS', 40) : undefined
+                        from: 0
+                        to: 180
+                        stepSize: 10
+                        suffixText: " s"
+
+                        onMoved: function () {
+                          iface.setScreenDimmerTimeout(value)
+                          settings.setValue('dimTimeoutS', value)
+                        }
+
+                        Layout.fillWidth: true
```

Comment:
Purely aesthetic, you can ignore, but it's nice to keep the layout margin padding stuff at the top.